AccessibilityInsightsForAndroidService/app/build.gradle

```diff
@@ -5,7 +5,6 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 28
-    buildToolsVersion '28.0.3'
```

Comment:
This version is too old for the updated gradle plugin to support. The error message from the plugin explaining this says that as of v4.x of the plugin, the plugin maintains its own pinned version to use by default, which it recommends we defer to.